run.sh

```diff
@@ -162,11 +162,14 @@ elif [[ "$PLATFORM" == perlmutter ]]; then
      # 64GM RAM per NUMA domain (256GB RAM per node)
      # 1 Ampere A100 GPU per NUMA domain (4 GPUs per node)
      # 40GB FB per GPU
+     USE_CUDA=1
```

Comment:
Do we need to set this during runtime? I think this is only relevant when building.

Reply:
`run.sh` seems to refer to it in several places
```
    if [[ "$USE_CUDA" == 1 ]]; then
        GPUS_PER_NODE="$(nvidia-smi -q | grep 'Attached GPUs' | awk '{print $4}')"
        # Leave 5% of the available framebuffer for other uses, but not less than 2GB
        FB_PER_GPU="$(nvidia-smi --format=csv,noheader,nounits --query-gpu=memory.total -i 0)"
        RESERVED_FB=$(( FB_PER_GPU / 20 ))
        RESERVED_FB=$(( RESERVED_FB > 2000 ? RESERVED_FB : 2000 ))
        FB_PER_GPU=$(( FB_PER_GPU - RESERVED_FB ))
```
and 
```
if [[ "$USE_CUDA" == 1 ]]; then
    NUM_GPUS=$(( GPUS_PER_NODE * $NODE_RATIO ))
fi
```

Reply:
You're right, I forgot about that.

Still, we shouldn't need to set it per-platform, since It is set globally by querying `legate --info` at the top of the file https://github.com/nv-legate/quickstart/blob/branch-23.07/run.sh#L93-L99 (and also we allow it to be overwritten by the user).

Reply:
https://github.com/nv-legate/quickstart/pull/7/commits/45dbf4cb9fd8dd839f6e7468713cd2c74388c03e